libs/antsi/src/font.rs

```diff
@@ -91,6 +89,13 @@ pub enum FontFamily {
 pub enum Underline {
     Single,
     Double,
+
+    // kitty + vte extension
+    Curly,
+    // kitty + vte extension
+    Dotted,
+    // kitty + vte extension
+    Dashed,
```

Comment:
It doesn't seem that these are in the standard but extensions? Not a blocker for this PR but we really should consider feature flags.

Reply:
Yes, but ANSI says that unrecognized escape sequences are to be ignored. Therefore even if not enabled, their formatting will still be ignored. I agree, tho that in the future (next PR), these will move into feature flags, while ignored they still provide the library user w/ more insight, by telling them: "this is an extension and not supported by every terminal emulator"